### ISSUE TITLE
Document how best to deploy to multiple EB applications and/or AWS accounts

### ISSUE THREAD

**Issue by msgilligan** [EvidentSolutions/gradle-beanstalk-plugin#19]
It seems the plugin is designed to publish to multiple EB environments within a single EB application in a single AWS account. It's not obvious from looking at the sample `beanstalk` DSL block on the README how to handle multiple applications or accounts.

I assume it would be possible by using Groovy variables in the build file, but it would be better if there was a standard/documented way to do it with the DSL and standard beanstalk Gradle tasks.

**Comment by komu** [comment]
Right, it should be possible to create a new instance of the task and pass custom properties for that.

Unfortunately I've never tested that and I don't even have an active beanstalk environment right now since I've been working on other things for couple of years. So I'd gladly merge a PR for documenting this, but I'm afraid I won't do it myself for the foreseeable future.

**Comment by msgilligan** [comment]
Thanks. I'll give it a try and also try to report back...

**Comment by rich5851** [comment]
I can confirm that this works.
